src/Hl7.Fhir.Specification/Validation/FpConstraintValidationExtensions.cs

```diff
@@ -36,6 +36,9 @@ public static OperationOutcome ValidateFp(this Validator v, ElementDefinition de
 
             foreach (var constraintElement in definition.Constraint)
             {
+                // Skip any best practice constraints until that setting is available
```

Comment:
https://github.com/ewoutkramer/fhir-net-api/issues/796
This is relevant to the above issue